source/contributing/how-to-contribute.md

```diff
@@ -1,77 +1,68 @@
-# How to Contribute
+# How to contribute
 
-This guide explains how you can contribute to Nix, Nix packages or
-NixOS.
+The Nix ecosystem is developed by many volunteers and a few paid developers, maintaining one of the largest open source software distributions in the world.
+Keeping it working and up to date would not be possible without your support.
 
-:::{note}
-Please also consider [contributing documentation](./documentation.md).
+This guide shows how you can contribute to {term}`Nix`, {term}`Nixpkgs` or {term}`NixOS`.
+
+## Getting started
+
+[Join our community communication platforms](https://nixos.org/community) to get in contact with other users and developers.
+Check out our [community teams](https://nixos.org/community/#governance-teams) if you're interested in a particular topic.
+
+All the source code and documentation is on [GitHub](https://github.com/), and you need a GitHub account to propose changes.
+Technical discussions happen in issue and pull request comments.
+
+:::{tip}
+If you are new to Nix, consider [contributing documentation](./documentation.md) first.
+
+This is where we need the most help and where it is the easiest to begin.
 :::
 
 ## Report an issue
 
-We can only fix issues that we know of, so please report any issue you
-encounter.
+We can only fix issues that we know of, so please report any issue you encounter.
 
-Issues with the **package manager Nix** (including its documentation)
-are reported at <https://github.com/NixOS/nix/issues>.
+Issues with {term}`Nix` (including the [Nix reference manual](https://nixos.org/manual/nix/stable)) are reported at <https://github.com/NixOS/nix/issues>.
 
-Issues with **specific packages or NixOS** (including its modules and
-documentation) are reported at <https://github.com/NixOS/nixpkgs/issues>.
+Issues with {term}`Nixpkgs` or {term}`NixOS` (including packages, configuration modules, the [Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable), and the [NixOS manual](https://nixos.org/manual/nixos/stable)) are reported at <https://github.com/NixOS/nixpkgs/issues>.
 
 Make sure that there is not already an open issue for your problem.
-Please follow the issue template and fill in all requested information
-as they help us solve the problem.
-
-You need a [GitHub] account for that.
-
-[github]: https://github.com/
+Please follow the issue template and fill in all requested information.
 
 ## Contribute to Nix
 
-The package manager Nix is mostly written in C++.
+The Nix is the cornerstone of the ecosystem, and mostly written in C++.
```

Comment:
```suggestion
The Nix package manager is the cornerstone of the ecosystem, and is mostly written in C++.
```

Reply:
```suggestion
Nix is the cornerstone of the ecosystem, and is mostly written in C++.
```

That was a genuine typo, forgot to remove the article. I try to get away from calling Nix a package manager because it's quite misleading in my opinion.